src/pykeen/models/base.py

```diff
@@ -567,7 +567,11 @@ def score_hrt_inverse(
         self,
         hrt_batch: torch.LongTensor,
     ) -> torch.FloatTensor:
-        """Score triples based on inverse triples, i.e., compute f(h,r,t) based on f(t,r_inv,h)."""
+        r"""Score triples based on inverse triples, i.e., compute f(h,r,t) based on f(t,r_inv,h).
+
+        When training with inverse relations, the model produces two (different) scores for a triple $(h,r,t) \in K$.
+        This function enables users to inspect the scores obtained by using the corresponding inverse triples.
```

Comment:
Is it already clear from the doc that `predict_all_heads` is using inverse relations by default? Otherwise, having this explanation here might look like it is only used here.

Reply:
I extented the dosctring of `predict_all_heads` in 9a50725af0c5b65f29ef2193bcbcf285ecb6283c to make it explicit.

Reply:
> I extented the dosctring of `predict_all_heads` in [9a50725](https://github.com/pykeen/pykeen/commit/9a50725af0c5b65f29ef2193bcbcf285ecb6283c) to make it explicit.

I just realized that it is already explained in https://github.com/pykeen/pykeen/blob/61065977fb6494f7d45ca5302b5e7f590f6f7035/src/pykeen/models/base.py#L616-L620.
However, it might be helpful to have this information directly in the docstring.